samples/fr-platform/values.yaml

```diff
@@ -1,11 +1,64 @@
-openam:
+frconfig:
+  domain: .svc.cluster.local
```

Comment:
@alexbugar I am specifying the domain value for frconfig here, in my values.yaml. I'm not sure what else you were suggesting I do. Care to elaborate?

Reply:
The suggestion is about allowing users the flexibility to come up with their own custom values.yaml file where a user can define their own "domain" as a global variable and also use a different "namespace" if they want to.